ipfx/bin/generate_se_input.py

```diff
@@ -14,11 +13,12 @@ def generate_se_input(cell_dir,
     se_input = dict()
 
     if specimen_id:
-        se_input['input_nwb_file'] = lq.get_input_nwb_file(specimen_id)
-        se_input['input_h5_file'] = lq.get_input_h5_file(specimen_id)
+        input_nwb_file = lq.get_input_nwb_file(specimen_id)
+        input_h5_file = lq.get_input_h5_file(specimen_id)
+        if input_h5_file:
+            se_input['input_h5_file'] = input_h5_file
 
-    elif input_nwb_file:
-        se_input['input_nwb_file'] = input_nwb_file
+    se_input['input_nwb_file'] = input_nwb_file
```

Comment:
The `elif input_nwb_file:` is still needed as both `specimen_id` and `input_nwb_file` can both be None.

Reply:
@t-b I have changed to using mutually exclusive arguments with `argparse`. Now, if `specimen_id` or `input_nwb_file` not provided, then the argparse will catch it before getting to this function.